src/components/ConversationIcon.vue

```diff
@@ -155,7 +154,7 @@ $icon-size: 44px;
 		width: $icon-size;
 		height: $icon-size;
 		line-height: $icon-size;
-		font-size: math.div($icon-size, 2);
+		font-size: $icon-size / 2;
```

Comment:
![Uploading Bildschirmfoto von 2022-01-18 10-30-17.png…]()

Reply:
Hmm, why do we get the warning in these two files only? We use this / everywhere outside calc()